src/js/components/product-select/ProductSelect.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+
+import { debounceProductsFetch } from 'utils/option-utils';
+import renderHandlingIcons from 'utils/product-handling-icons';
+import Select from 'utils/Select';
+
+const Option = option => (
+  <strong style={{ color: option.color || 'black' }} className="d-flex align-items-center">
+    {option.label}
+  &nbsp;
+    {renderHandlingIcons(option.handlingIcons)}
+  </strong>);
+
+const SelectedValue = option => (
+  <span className="d-flex align-items-center">
+    <span className="text-truncate">{option.label}</span>
+    &nbsp;{renderHandlingIcons(option ? option.handlingIcons : [])}
```

Comment:
just want to show a different option:
```
option?.handlingIcons ?? []
```

Reply:
yeah sure, I just copied and pasted this chunk of code so didn't really notice that